src/Common/MoistThermodynamics/states.jl

```diff
@@ -78,18 +78,36 @@ struct PhaseEquil{FT, PS} <: ThermodynamicState{FT}
     "temperature: computed via [`saturation_adjustment`](@ref)"
     T::FT
 end
+
+"""
+    PhaseEquil
+
+Moist thermodynamic phase, given
+ - `param_set` an `AbstractParameterSet`, see the [`MoistThermodynamics`](@ref) for more details
+ - `e_int` internal energy
+ - `ρ` density
+ - `q_tot` total specific humidity
+
+and, optionally
+
+ - `maxiter` maximum iterations for saturation adjustment
+ - `temperature_tol` estimated temperature tolerance for saturation adjustment
+ - `sat_adjust` function pointer to particular saturation adjustment function
+"""
 function PhaseEquil(
     param_set::APS,
     e_int::FT,
     ρ::FT,
     q_tot::FT,
-    maxiter::Int = 3,
-    tol::FT = FT(1e-1),
+    maxiter::Int = 5,
```

Comment:
Do we need 'maxiter' to be 5 in the tests. Or does 3 suffice?

Reply:
Turns out we need 6 iterations at `temperature_tol = 1e-1`